util/test/testUtil.h

```diff
@@ -44,7 +45,7 @@ void valuecheck(const Eigen::DenseBase<DerivedA>& a, const Eigen::DenseBase<Deri
 
 void valuecheck(double a, double b, double tolerance = 1e-8)
 {
-  if (abs(a - b) > tolerance) {
+  if (std::abs(a - b) > tolerance) {
```

Comment:
you're right, that's definitely bad. it would be cleaner to have drakeUtil drakeMexUtil as separate bits.  (@psiorx has some more drakeMex helpers in his open pull request)

On Apr 10, 2015, at 8:13 AM, Twan Koolen <notifications@github.com> wrote:

> In util/test/testUtil.h:
> 
> > @@ -44,7 +45,7 @@ void valuecheck(const Eigen::DenseBase<DerivedA>& a, const Eigen::DenseBase<Deri
> >  
> >  void valuecheck(double a, double b, double tolerance = 1e-8)
> >  {
> > -  if (abs(a - b) > tolerance) {
> > +  if (std::abs(a - b) > tolerance) {
> so drakeUtil is currently more of a drakeMexUtil (apart from some methods that don't look like they belong anymore). How do you feel about including mex stuff in source files that would otherwise not need it?
> 
> —
> Reply to this email directly or view it on GitHub.
>